core/modules/qmeta/QMeta.h

```diff
@@ -89,20 +90,21 @@ class QMeta  {
      *  @param name:  Czar ID, non-negative number.
      *  @param active:  new value if active flag.
      */
-    virtual void setCzarActive(int czarId, bool active) = 0;
+    virtual void setCzarActive(CzarId czarId, bool active) = 0;
 
     /**
      *  @brief Register new query.
      *
      *  This method will throw if czar ID is not known.
      *
-     *  @param qInfo:  Query info instance, time members (submitted/collected) are ignored.
+     *  @param qInfo:  Query info instance, time members (submitted/collected) and query status
+     *                 are ignored.
```

Comment:
Looks like "collected" was changed to "completed", and the docs here should also be updated.